src/components/BannerWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/function-component-definition */
```

Comment:
Don't disable eslint rules. They are there for a reason. Please follow function declaration format for the react components. I also recommend having a read through here: https://suwanna.atlassian.net/wiki/spaces/SD/pages/2031638/Style+Guide